### ISSUE TITLE
Tensorflowjs model

### ISSUE THREAD

**Issue by camhart** [JaidedAI/EasyOCR#73]
I can't seem to find much online about converting pytorch models over to tensorflowjs.  I'm looking for a good scene text detection model primarily for use on screenshots--so the text will be very clear.

I'd even be willing to donate a little if the accuracy was sufficiently high.

**Comment by rkcosmos** [comment]
I'll mark this as "Application" in case someone want to create a project.

**Comment by zacharywhitley** [comment]
You need to convert them to ONNX and use onnx.js
